src/test/unit/StakeRegistryUnit.t.sol

```diff
@@ -190,7 +191,226 @@ contract StakeRegistryUnitTests is Test {
         stakeRegistry.registerOperator(defaultOperator, defaultOperatorId, quorumNumbers);
     }
 
-    function testOperatorStakeUpdate_Valid(
+    function testRegisterFirstOperator_Valid(
+        uint256 quorumBitmap,
+        uint80[] memory stakesForQuorum
+    ) public {
+
+        uint96[] memory paddedStakesForQuorum = _registerOperatorValid(defaultOperator, defaultOperatorId, quorumBitmap, stakesForQuorum);
+
+        uint8 quorumNumberIndex = 0;
+        for (uint8 i = 0; i < 192; i++) {
+            if (quorumBitmap >> i & 1 == 1) {
```

Comment:
should we make a utility getter function for this *on the stake registry* that is just something like
```
function isOperatorRegisteredForQuorum(address operator, uint8 quorumNumber) external view returns (uint8) {
...
}
```
?
I guess we could have a version that takes in a blockNumber or index too?
This could also be part of the BLSOperatorStateRetriever, I suppose; I'm just seeing you repeating the code block and realizing it could be useful!

Reply:
i'm avoiding messing with the stake registry unit tests for now, planning for them to get done in another PR

Reply:
OK, please add this as part of that PR then, as I think it would be a good addition